File: projekat/RentApp/Models/Entities/Service.cs
using System.Collections.Generic;

namespace RentApp.Models.Entities
{
    public class Service
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string LogoUrl { get; set; }
        public string Email { get; set; }
        public string Description { get; set; }
        public bool Approved { get; set; }

        public ICollection<Branch> Branches { get; set; }
        public ICollection<Rating> Ratings { get; set; }
        public ICollection<Vehicle> Vehicles { get; set; }
    }
}

File: projekat/AngularApp/Web2Projekat2/src/app/model/service.ts
export class Service
{
    public id: number
        public name: string
        public logoUrl: string
        public email: string
        public description:string
        public approved: boolean

        public branches: object[]
        public ratings: object[]
        public vehicles: object[]
  constructor()
  {
      this.name="";
      this.logoUrl="";
      this.email="";
      this.description="";
      this.approved=false;

      this.branches;
      this.ratings;
      this.vehicles;
  }
}